test/e2e/providers/vsphere/vsphere.go

```diff
@@ -1,8 +1,11 @@
 package vsphere
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Looks like this is already imported as `meta`, below. You should use that one instead